envoy-control-runner/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/debug/SnapshotDebugController.kt

```diff
@@ -51,6 +55,25 @@ class SnapshotDebugController(controlPlane: ControlPlane) {
         }
     }
 
+    @GetMapping("/snapshot/global")
+    fun globalSnapshot(@RequestParam xds: Boolean?): ResponseEntity<SnapshotDebugInfo> {
```

Comment:
I think it should be tested, just like other endpoints in this class.

Reply:
Tests added